src/models/local/LocalAppController.js

```diff
@@ -22,18 +21,8 @@ export default class LocalAppController extends LocalBaseController {
         ? await StdlibInstaller.call(stdlibNameVersion)
         : new Stdlib(stdlibNameVersion)
 
-      this.packageData.stdlib = {
-        name: stdlib.getName(),
-        version: stdlib.getVersion()
-      }
+      const { name, version } = stdlib
+      this.packageFile.stdlib = { name, version }
```

Comment:
If we are restricting the stdlib object to only have name and version, I'd do that filtering in the `packageFile` object rather than here.